meet_balls_app/enums.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum, IntEnum
+
+
+class BallGame(Enum):
+    SOCCER = 'Soccer'
+    BASKETBALL = 'Basketball'
+    VOLLEYBALL = 'Volleyball'
+    BASEBALL = 'Baseball'
+    TENNIS = 'Tennis'
+    RUGBY = 'Rugby'
+    GOLF = 'Golf'
+    CRICKET = 'Cricket'
+    HANDBALL = 'Handball'
+
+    @classmethod
+    def BALL_GAME_CHOICES(cls):
+        return [(tag.value, tag.name) for tag in cls]
+
+
+class Rating(IntEnum):
```

Comment:
How is this one being used?

Reply:
As Yuval suggested we added this instead in the models.py file:

class BallGame(models.TextChoices):
    Soccer = 'Soccer', 'Soccer'
    Basketball = 'Basketball', 'Basketball'
    Volleyball = 'Volleyball', 'Volleyball'
    Baseball = 'Baseball', 'Baseball'
    Tennis = 'Tennis', 'Tennis'
    Rugby = 'Rugby', 'Rugby'
    Golf = 'Golf', 'Golf'
    Cricket = 'Cricket', 'Cricket'
    Handball = 'Handball', 'Handball'

Reply:
@amitKaplansky Is that about the rating or about the game choices?

Reply:
game choices, @MaayanMashhadi and I used it in the GameEvent entity

Reply:
My comment was regarding `class Rating`.

Reply:
@amitKaplansky Where is that entity that you mention?

Reply:
still working on this entity,  so I didn't add the PR

Reply:
I think line 20 can be deleted for now @Danielsio

Reply:
> How is this one being used?

It's not being used currently, Removed.

Reply:
> As Yuval suggested we added this instead in the models.py file:
> 
> class BallGame(models.TextChoices): Soccer = 'Soccer', 'Soccer' Basketball = 'Basketball', 'Basketball' Volleyball = 'Volleyball', 'Volleyball' Baseball = 'Baseball', 'Baseball' Tennis = 'Tennis', 'Tennis' Rugby = 'Rugby', 'Rugby' Golf = 'Golf', 'Golf' Cricket = 'Cricket', 'Cricket' Handball = 'Handball', 'Handball'

Thanks you for the comment, that works.